extensions/applicationinsights-dependencies-js/src/ajax.ts

```diff
@@ -541,6 +542,7 @@ export class AjaxMonitor implements ITelemetryPlugin, IDependenciesPlugin, IInst
                     id: ajaxData.id,
                     target: ajaxData.getAbsoluteUrl(),
                     type: ajaxData.getPathName(),
+                    requestType: "Fetch",
```

Comment:
So Type and requestType are different things?, this doesn't look good in Ajax you sue type and in Fetch calls you use both requestType and type, is this new field documented anywhere?

Reply:
type is existing field. Would prefer to name this type. The default value of `requestType` is `"Ajax"`, but it should probably be explicitly set here.

Reply:
Also I think `type` is old `commandName`